common/src/main/java/com/wynntils/features/inventory/UnidentifiedItemIconFeature.java

```diff
@@ -11,39 +11,62 @@
 import com.wynntils.core.persisted.config.Category;
 import com.wynntils.core.persisted.config.Config;
 import com.wynntils.core.persisted.config.ConfigCategory;
+import com.wynntils.core.text.StyledText;
 import com.wynntils.mc.event.HotbarSlotRenderEvent;
 import com.wynntils.mc.event.SlotRenderEvent;
 import com.wynntils.models.gear.type.GearType;
+import com.wynntils.models.items.WynnItem;
 import com.wynntils.models.items.items.game.GearBoxItem;
+import com.wynntils.models.items.items.game.GearItem;
+import com.wynntils.utils.colors.CommonColors;
+import com.wynntils.utils.render.FontRenderer;
 import com.wynntils.utils.render.RenderUtils;
 import com.wynntils.utils.render.Texture;
+import com.wynntils.utils.render.type.HorizontalAlignment;
+import com.wynntils.utils.render.type.TextShadow;
+import com.wynntils.utils.render.type.VerticalAlignment;
 import com.wynntils.utils.type.Pair;
+import java.util.EnumMap;
 import java.util.Map;
 import java.util.Optional;
+import java.util.function.Function;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
 import net.minecraft.world.item.ItemStack;
 import net.neoforged.bus.api.SubscribeEvent;
 
 @ConfigCategory(Category.INVENTORY)
 public class UnidentifiedItemIconFeature extends Feature {
-    private static final Map<GearType, Pair<Integer, Integer>> TEXTURE_COORDS = Map.ofEntries(
-            Map.entry(GearType.SPEAR, Pair.of(16 * 1, 16 * 1)),
-            Map.entry(GearType.WAND, Pair.of(16 * 0, 16 * 1)),
-            Map.entry(GearType.DAGGER, Pair.of(16 * 2, 16 * 1)),
-            Map.entry(GearType.BOW, Pair.of(16 * 3, 16 * 1)),
-            Map.entry(GearType.RELIK, Pair.of(16 * 0, 16 * 2)),
-            Map.entry(GearType.RING, Pair.of(16 * 1, 16 * 2)),
-            Map.entry(GearType.BRACELET, Pair.of(16 * 2, 16 * 2)),
-            Map.entry(GearType.NECKLACE, Pair.of(16 * 3, 16 * 2)),
-            Map.entry(GearType.HELMET, Pair.of(16 * 0, 16 * 0)),
-            Map.entry(GearType.CHESTPLATE, Pair.of(16 * 1, 16 * 0)),
-            Map.entry(GearType.LEGGINGS, Pair.of(16 * 2, 16 * 0)),
-            Map.entry(GearType.BOOTS, Pair.of(16 * 3, 16 * 0)),
-            Map.entry(GearType.MASTERY_TOME, Pair.of(16 * 0, 16 * 3)),
-            Map.entry(GearType.CHARM, Pair.of(16 * 1, 16 * 3)));
+    private static final StyledText QUESTION_MARK_TEXT =
+            StyledText.fromComponent(Component.literal("?").withStyle(ChatFormatting.BOLD));
+    private static final Map<GearType, Pair<Integer, Integer>> TEXTURE_COORDS = new EnumMap<>(GearType.class);
+
+    static {
```

Comment:
Did you do this change out of stylistic reasons, or is there some other rationale? 

While technically very similar in the resulting byte code, I strongly dislike `static` blocks. They make for surprising code behavior imho. In contrast, constructing the map at the point of the `final` declaration makes it very clear what it contains.

Reply:
I see it's not an EnumMap, perhaps that was the reason? If there wasn't any problem with using a regular HashMap, then probably should be reverted.

Reply:
Yeah... Enum maps are technically somewhat faster, but sacrificing code legibility for performance should only be done if it has a measured impact.

Reply:
Then, just wrap it in an Enum map. GC during init is fine. Although the positive gain from this change is likely minimal. Don't get us wrong, we like performant code, but not at the expense of readability and maintainability.